strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/yaml/configuration/repository/MavenRepositoryConfigurationDto.java

```diff
@@ -22,6 +23,8 @@
     // defaults to once daily at 2 am
     private String cronExpression = "0 0 2 * * ?";
 
+    private String metadataExpirationStrategy = MetadataExpirationStrategyType.CHECKSUM.name();
```

Comment:
`CHECKSUM.name()`

it should be 

`CHECKSUM.desribe()` since we operate on enum `strategy` field, not `name`